cmd/configtxgen/main_test.go

```diff
@@ -34,7 +35,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestInspectMissing(t *testing.T) {
-	assert.Error(t, doInspectBlock("NonSenseBlockFileThatDoesn'tActuallyExist"), "Missing block")
+	assert.EqualError(t, doInspectBlock("NonSenseBlockFileThatDoesn'tActuallyExist"), "Could not read block NonSenseBlockFileThatDoesn'tActuallyExist")
```

Comment:
I know you didn't introduce these error messages, but per the general Go programming conventions, error strings should not start capitalized https://github.com/golang/go/wiki/CodeReviewComments#error-strings would you be able to go through the production code as well as the tests in this path and update the error strings accordingly? There's a lot of places in Fabric where we don't properly follow this convention but might as well start here since we're touching this path.